target/linux/ramips/rt3883/base-files/etc/board.d/02_network

```diff
@@ -27,6 +27,8 @@ ramips_setup_interfaces()
 		;;
 	loewe,wmdr-143n|\
 	omnima,hpm)
+		ucidef_add_switch "switch0"
+		ucidef_add_switch_attr "switch0" "enable" "false"
```

Comment:
rt3883 doesn't have a built-in switch. This isn't needed.

Reply:
Oh, sorry, I didn't know that. Then we should revert a1693bf626f8...